File: 프로그래머스/Lv.2/42584. 주식가격/주식가격.js
function solution(prices) {
    var answer = [];
    for (let i = 0; i<prices.length; i++){
        let temp = 0
        for(let j=i+1; j<prices.length; j++){
            if(prices[i]<=prices[j]){
                temp++;
            }
            else{
                temp++;
                break
                }
            
        }
        answer.push(temp);
    }
    return answer;
}

File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = [0] * len(prices)

    for i in range(0,len(prices)):
        count = 1
        for j in range(i+1, len(prices)):
            if(prices[i]<= prices[j] and j != len(prices)-1):
                count+=1
                continue
            
            answer[i] = count
            break
            
            
        
    return answer